cpp/example_code/glue/README.md

```diff
@@ -107,6 +108,30 @@ This example shows you how to do the following:
 - List information about job runs, view transformed data, and clean up resources.
 
 <!--custom.scenario_prereqs.glue_Scenario_GetStartedCrawlersJobs.start-->
+
+You can create the resources needed for the example using the CDK script [resources/cdk/glue_role_bucket](../../../resources/cdk/glue_role_bucket).
+
+Note: The AWS CDK and npm must be installed to use the script.
+
+From the [resources/cdk/glue_role_bucket](../../../resources/cdk/glue_role_bucket) folder, run the commands:
+```bash
+npm install
+cdk deploy
+```
+
+Use the following outputs in the example.
+
+```bash
+doc-example-glue-scenario-stack.BucketName = doc-example-glue-scenario-s-docexampleglue1234567-890abcdef
+doc-example-glue-scenario-stack.RoleName = AWSGlueServiceRole-DocExample
```

Comment:
Will this work for any customer account? (I guess it's the role created in the CDK, above)

Reply:
It is a follow on the MVP.
I am giving them documentation on CDK outputs.

Reply:
This is the expected output of the CDK script? I think I'm missing something, so line 122 might need a bit more commentary on what this block is meant to be?